.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: rust
 rust:
-- nightly
+- stable 
```

Comment:
There's a trailing space here. Seems to not affect anything, but it should probably not be there.

Reply:
updated it